src/Messenger/Stamp/AzureBrokerPropertiesStamp.php

```diff
@@ -107,7 +107,15 @@ public function __construct(
     public static function createFromResponse(ResponseInterface $response): self
     {
         $header = $response->getHeaders()['brokerproperties'][0] ?? '';
+        return AzureBrokerPropertiesStamp::createFromJson($header);
```

Comment:
If you can change the call to `self::` that'd be perfect.

Reply:
it is done.